src/olympia/devhub/tests/test_views.py

```diff
@@ -2360,12 +2360,7 @@ def test_get_verification_timedout(self, mock_check_emails):
             assert 'It is taking longer than expected' in doc.text()
             assert 'Send another email' in doc.text()
 
-            assert 'Having trouble?' in doc.text()
-            support_link = doc('a:contains("contact AMO Admins")')
-            assert f'mailto:{settings.SUPPORT_EMAIL}' in support_link.attr('href')
-            assert '?subject=Suppressed Email verification' in support_link.attr('href')
-            assert 'I have a suppressed email' in support_link.attr('href')
-            assert self.user_profile.email in support_link.attr('href')
+            assert 'If you encounter issues' in doc.text()
```

Comment:
Please add an assertion for the full relative url.

"/the-entire#url?with=query,params=foo"

Reply:
605c1ac00bd